bisqapps/shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/screens/URLScreen.kt

```diff
@@ -0,0 +1,153 @@
+
+private lateinit var textState: MutableState<String>
+
+@OptIn(ExperimentalResourceApi::class)
+@Composable
+fun URLScreen(
+    rootNavController: NavController
+) {
+    textState = remember { mutableStateOf("") }
+    val isConnected by remember { mutableStateOf(false) }
+
+    BisqScrollLayout() {
+        BisqLogo()
+        Spacer(modifier = Modifier.height(24.dp))
+        Column(
+            verticalArrangement = Arrangement.SpaceBetween,
+            modifier = Modifier.fillMaxSize().padding(horizontal = 24.dp)
+        ) {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                BisqText.baseRegular(
+                    text = "Bisq URL",
+                    color = BisqTheme.colors.light1,
+                )
+                Image(painterResource(Res.drawable.icon_question_mark), "Question mark")
+            }
+
+            MaterialTextField(textState.value, onValueChanged = { textState.value = it })
+
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween
+            ) {
+                BisqButton(
+                    text = "Paste",
+                    onClick = {},
+                    backgroundColor = BisqTheme.colors.dark5,
+                    color = BisqTheme.colors.light1,
+                    //leftIcon=Image(painterResource(Res.drawable.icon_copy), "Copy button")
+                )
+
+                BisqButton(
+                    text = "Scan",
+                    onClick = {},
+                    //leftIcon=Image(painterResource(Res.drawable.icon_qr), "Scan button")
+                )
+            }
+            Spacer(modifier = Modifier.height(36.dp))
+            BisqText.baseRegular(
+                text = "STATUS",
+                color = BisqTheme.colors.grey2,
+            )
+            Spacer(modifier = Modifier.height(12.dp))
+            Row(verticalAlignment = Alignment.CenterVertically) {
+                BisqText.largeRegular(
+                    text = if (isConnected) "Connected" else "Not Connected",
+                    color = BisqTheme.colors.light1,
+                )
+                Spacer(modifier = Modifier.width(12.dp))
+                BisqText.baseRegular(
+                    text = "",
+                    modifier = Modifier.clip(
+                        RoundedCornerShape(5.dp)
+                    ).background(color = if (isConnected) BisqTheme.colors.primary else BisqTheme.colors.danger)
+                        .size(10.dp),
+                )
+            }
+        }
+
+        Spacer(modifier = Modifier.height(56.dp))
+
+        var visible by remember {
+            mutableStateOf(false)
+        }
+
+        if (!visible) {
+            BisqButton(
+                text = "Test Connection",

```

Comment:
shouldn't we be using the Locale strings instead of hardcoding here? Just putting a comment for this one but there are several cases

Reply:
Yeah. Implemented i18n only in SplashScreen.
Wanted to know if usage of lyricist library for i18n is fine, before implementing it everywhere.

Will implement it across now.

I will try to get the string resource names from bisq2 repo and re-use same names here.

Reply:
we discussed this on matrix, no need to invest time in i18n is not part of the MVP, we will support only English initially.

just wrap it up and don't spend much time on it I would suggest